### PR TITLE
feat: [es] Create /docs/contributing/pull-requests.md

### PR DESCRIPTION
**Pull request by TineoC**
**Description**
This pull request adds the Spanish localization of the /docs/contributing/new-content page on the OpenTelemetry website. The translation includes all sections from the original English document, maintaining the format and structure to ensure consistency with other localized sections.

**Changes made:**
* Created the pull-requests.md file in the docs/es/contributing/ directory.
* Translated the original content, preserving technical semantics and reference links.
* Minor adjustments were made to the Markdown content for improved display.

**References**
This work is related to issue [#5205](https://github.com/open-telemetry/opentelemetry.io/issues/5205), which tracks the Spanish localization of the contributing documentation folder.
This work is related to pull request [#5270](https://github.com/open-telemetry/opentelemetry.io/pull/5270), which was closed because I used merge instead of rebase.

### CODE REVIEW

**Comment by TineoC** [comment]
@fabrideci

File: content/es/docs/contributing/pull-requests.md [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+  editor de código.
+weight: 2
+default_lang_commit: f724c15be360e5059fb89e696d9a5cc8d00496f6
+cSpell:ignore: aplícala vincúlalos solucionándolas
+---
+
+Para contribuir con nuevas páginas de contenido o mejorar las páginas de
+contenido existentes, abre un Pull Request (PR):
+
+- Si tu cambio es pequeño, o ya estás familiarizado con Git, lee
```
Comment (mx-psi) [inline content/es/docs/contributing/pull-requests.md:14]: El original es "you’re unfamiliar with Git"
```suggestion
- Si tu cambio es pequeño, o no estás familiarizado con Git, lee
```

File: content/es/docs/contributing/pull-requests.md [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+  [Editar con GitHub](#changes-using-github) para aprender a editar la página
+  desde la interfaz de GitHub.
+- Si tus cambio es mayor, lee [Editar desde una rama local](#fork-the-repo) para
```
Comment (mx-psi) [inline content/es/docs/contributing/pull-requests.md:17]: "mayor" no suena idiomático, yo usaría algo como esto:
```suggestion
- Si tu cambio es más complejo, lee [Editar desde una rama local](#fork-the-repo) para
```

File: content/es/docs/contributing/pull-requests.md [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+  aprender cómo realizar cambios desde tu editor de código.
+
+{{% alert title="Tip" %}}
```
Comment (mx-psi) [inline content/es/docs/contributing/pull-requests.md:20]:
```suggestion
{{% alert title="Consejo" %}}
```

File: content/es/docs/contributing/pull-requests.md [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+subgraph tasks[Editando en GitHub]
+direction TB
+    0[ ] -.-
+    1[Edita esta página] --> 2[Usa el editor de markdown en GitHub<br>para hacer tus cambios]
+    2 --> 3[agrega tus propuesta de cambios]
+
+end
+subgraph tasks2[ ]
+direction TB
+4[Selecciona tus archivos a cambiar] --> 5[Haz click en Crear Pull Request] --> 6[Llena los cambios en Abriendo un Pull Request]
```
Comment (mx-psi) [inline content/es/docs/contributing/pull-requests.md:70]:
```suggestion
4[Selecciona los archivos a cambiar] --> 5[Haz click en Crear Pull Request] --> 6[Llena los cambios en Abriendo un Pull Request]
```

File: content/es/docs/contributing/pull-requests.md [hunk 5]
```diff
@@ -0,0 +1,511 @@
...
+continuación.
+
+```mermaid
+flowchart LR
+A([fa:fa-user Nuevo<br>Contribuyente]) --- id1[(open-telemetry/opentelemetry.io<br>GitHub)]
+subgraph tasks[Editando en GitHub]
+direction TB
+    0[ ] -.-
+    1[Edita esta página] --> 2[Usa el editor de markdown en GitHub<br>para hacer tus cambios]
+    2 --> 3[agrega tus propuesta de cambios]
```
Comment (mx-psi) [inline content/es/docs/contributing/pull-requests.md:65]:
```suggestion
    2 --> 3[Agrega tus propuesta de cambios]
```

File: content/es/docs/contributing/pull-requests.md [hunk 6]
```diff
@@ -0,0 +1,511 @@
...
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,C,D,E,F,G,H grey
+class S,T spacewhite
+class first,second white
+```
+
+_Figure 1. Contribuyendo con nuevo contenido._
```
Comment (mx-psi) [inline content/es/docs/contributing/pull-requests.md:50]:
```suggestion
_Figura 1. Contribuyendo con nuevo contenido._
```

File: content/es/docs/contributing/pull-requests.md [hunk 7]
```diff
@@ -0,0 +1,511 @@
...
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef k8s fill:#326ce5,stroke:#fff,stroke-width:1px,color:#fff;
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,1,2,3,4,5,6,7 grey
+class 0 spacewhite
+class tasks,tasks2 white
+class id1 k8s
+```
+
+_Figure 2. Pasos para abrir un Pull Request desde GitHub._
```
Comment (mx-psi) [inline content/es/docs/contributing/pull-requests.md:86]:
```suggestion
_Figura 2. Pasos para abrir un Pull Request desde GitHub._
```

**Review by theletterf — APPROVED** [review]

**Pull request merged**